src/client/components/VesselTeaser.js

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+
+function Vessel({ id, vesselName, departureFrom, arrivesAt, etd, eta }) {
```

Comment:
Rename to `VesselTeaser`. Check to make sure that's consistent throughout the PR.

Reply:
renamed to VesselTeaser